core/tests/protractor/accessibility.js

```diff
@@ -41,6 +67,312 @@ describe('screenreader and keyboard user accessibility features', function() {
         .getAttribute('id'));
   });
 
+  it('should test navigation shortcuts for the community-library page',
+    async function() {
+      await browser.get('community-library');
+      await waitForUrlRedirection('community-library');
```

Comment:
The fact that the correct URL has loaded does not necessarily mean the page has loaded. It'd be more reliable to wait for elements on the page. Ditto for the rest of these tests

Reply:
@U8NWXD Are you suggesting I don't use `waitForUrlRedirection('community-library'); ` and instead use some sort of `waitForElementsToLoad` function on the page.

If so should I disregard these two comments above:
https://github.com/oppia/oppia/pull/10036#discussion_r460632437
https://github.com/oppia/oppia/pull/10036#discussion_r460632926

and just delete the `waitForUrlRedirection()` function

Reply:
Does `await waitFor.pageToFullyLoad();` work here?

Reply:
@sagangwee No It only works on pages with the loading indicator @U8NWXD ping

Reply:
That makes sense. I now agree that `waitFor.urlRedirection` is fine here

Reply:
Actually, `waitFor.urlRedirection` works only when we are waiting for the destination page to load after you use the shortcut. When we are waiting for the page from which the shortcut will be triggered, though, we need the page to have loaded so that the code that handles the shortcut is available. To see why this could be problematic, imagine that at the start of the tests we navigate to the getting started page. Now let's say the URL has loaded but the page hasn't. Then we would send the keyboard shortcut, but since the page hasn't loaded, it won't be handled properly.

Reply:
@U8NWXD Hmm so in your example are you saying that 
1.) There's a chance the keyboard shortcuts wouldn't do anything 
2.) Or that the previous page's navigation shortcuts would be tested instead of the one we want?

Because regarding the first point, it's been handling the keyboard shortcuts very consistently so far, so I don't think that should be an issue.

Regarding the second possibility, I investigated a little and I did a manual test where I attempted to use the keyboard shortcuts while the page was redirecting, it could not redirect them at all. 

I also ran a protractor test where I attempted to use keyboard shortcuts on the admin page (where the keyboard shortcuts are disabled).  If it was using the previous pages shortcuts instead, it could possibly pass this test.

```
    await browser.get(GET_STARTED_URL);
    await waitFor.urlRedirection('http://localhost:9001/get-started');

    await holdCtrlAndPressKey('0');
    await waitFor.urlRedirection(GET_STARTED_URL);

    **await browser.get('admin');
    await waitFor.urlRedirection('http://localhost:9001/admin');**

    await holdCtrlAndPressKey('0');
    await waitFor.urlRedirection(GET_STARTED_URL);
```

And it correctly got stuck on that page with the output

```
1) screenreader and keyboard user accessibility features should test the navigation shortcut ctrl+0
  - Failed: URL redirection took too long
  Wait timed out after 15010ms
```

Reply:
@jimbyo I am concerned about (1). As I understand it, the tests now are assuming that after the URL has loaded, the page's redirect-handling code loads before the next line in the test executes. This is a race condition that we should avoid if at all possible, even if in practice the test looks reliable. My reasoning here is that this test is going to be run dozens if not hundreds of times a month, so even if it only flakes 1% of the time, those will be flakes that block people's PRs from going in.

Right now our e2e tests are so flaky that merging this PR probably wouldn't make much of a difference, but race conditions like this are hard to find (aside: this is why it's taking us so long to fix the e2e tests). Eventually we'll get the flakiness down far enough where we're trying to fix race conditions like this one, at which point we'll wish we had fixed it before the PR merged. It's so much easier to make sure the code is robust at the start than try to find the race condition later that I think it's worth being really careful about it. That's why I'm being so nit-picky.

Since we just care that the page has loaded and not that any particular elements are available, you could wait for an element that's present on all Oppia pages. A cursory glance suggests that the `div` with class `oppia-content-container` might work? Then this change is just pasting the same `waitFor` statement over and over, so it shouldn't be too complicated I hope?